src/Angor/Avalonia/AngorApp/Sections/Portfolio/Manage/RecoveryStateViewModel.cs

```diff
@@ -37,22 +39,25 @@ public RecoveryStateViewModel(WalletId WalletId, InvestorProjectRecoveryDto dto,
 
         BatchAction = CreateBatchCommand(this);
     }
-    
-     private IEnhancedCommand<Maybe<Guid>> CreateBatchCommand(RecoveryStateViewModel recoveryStateViewModel)
+
+    private IEnhancedCommand<Maybe<Guid>> CreateBatchCommand(RecoveryStateViewModel recoveryStateViewModel)
     {
-        if (recoveryStateViewModel.CanRecover)
+        if (recoveryStateViewModel.CanSpendEndOfProjectOrThreshold)
         {
-            return ReactiveCommand.CreateFromTask(() => Recover(recoveryStateViewModel)).Enhance("Recover Funds");
+            var buttonText = recoveryStateViewModel.IsBelowPenaltyThreshold
```

Comment:
I think the text here should not be (below threshold) it hard for the user to understand what that means, perhaps spend funds or withdraw now etc.